diagnostic.md

```diff
@@ -21,13 +21,19 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+Comfort:2
+clarity:3
+I used google and the lessons to help
+
+curl --include --request GET http://www.example.com/donuts
+
 ```
 
 Write an AJAX request to retrieve a list of **all donuts** on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = $.get(http://www.example.com/donuts);
+getDonuts;
 ```
```

Comment:
This is *technically* almost correct, but we want you to use the .ajax method, as it is easier to configure.
Even still, doing it this way without a return, wouldn't do anything.

```js
let getDonuts = function () {
  return $.ajax({
    url: 'http://www.example.com/donuts',
    method: 'GET',
  });
};
```